src/consts.rs

```diff
@@ -5,17 +5,17 @@
 // Allow dead code in this module, since it's all packet consts anyways.
 #![allow(dead_code)]
 
-pub const HID_RPT_SIZE: usize = 64;
```

Comment:
This is used by all the other platforms as well, so we'll have to find/replace across the repo.